listenbrainz/webserver/static/js/src/stats/BarDualTone.tsx

```diff
@@ -148,14 +148,14 @@ export default function BarDualTone(props: BarDualToneProps) {
     let dateString: string;
     let listenCount: number;
     if (id === "lastRangeCount") {
-      const lastRangeDate = new Date(datum.lastRangeTs! * 1000);
+      const lastRangeDate = new Date((datum.lastRangeTs ?? 0) * 1000);
```

Comment:
These should be defined but both props are marked optional because they depend on the `id` check above (if/else).
But TS is right, better play it safe and default to a number